crates/egui/src/containers/scroll_area.rs

```diff
@@ -608,7 +629,7 @@ impl Prepared {
 
         if content_is_too_large[0] || content_is_too_large[1] {
             // Drag contents to scroll (for touch screens mostly):
-            let sense = if self.scrolling_enabled {
+            let sense = if self.drag_to_scroll_enabled {
```

Comment:
Shouldn't it check both things ?
let sense = if self.scrolling_enabled && self.drag_to_scroll_enabled {

Reply:
Yes, it should